packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/utils/schema.js

```diff
@@ -224,7 +226,13 @@ const createYupSchemaAttribute = (type, validations, options) => {
           return true;
         }
 
-        if (isNumber(value) || isNull(value) || isObject(value) || isArray(value)) {
+        if (
+          isValidJSONString(value) ||
+          isNumber(value) ||
+          isNull(value) ||
+          isObject(value) ||
```

Comment:
`isObject` returns true for arrays so you could either use `isPlainObject` && `Array.isArray` or just `isObject`

Reply:
Also can you split the imports of lodash function as follows:

`import get from 'lodash/get'; ` as it treeshakes better?